site/content/en/v1.4/tasks/traffic/zone-aware-routing.md

```diff
@@ -0,0 +1,218 @@
+* There must be at least two valid topology zones for scheduling.
+* {{< boilerplate prerequisites >}}
+
+## Configuration
+
+### Kubernetes Service
+Create the example Kubernetes Service with either topology aware routing or traffic distribution enabled.
+
+#### Topology Aware Routing
+{{< tabpane text=true >}}
+{{% tab header="Apply from stdin" %}}
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.kubernetes.io/topology-mode: Auto
+  name: zone-aware-routing-backend
+  labels:
+    app: zone-aware-routing-backend
+    service: zone-aware-routing-backend
+spec:
+  ports:
+    - name: http
+      port: 3000
+      targetPort: 3000
+  selector:
+    app: zone-aware-routing-backend
+EOF
+```
+{{% /tab %}}
+{{% tab header="Apply from file" %}}
+Save and apply the below service manifest to your cluster:
+```yaml
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.kubernetes.io/topology-mode: Auto
+  name: zone-aware-routing-backend
+  labels:
+    app: zone-aware-routing-backend
+    service: zone-aware-routing-backend
+spec:
+  ports:
+    - name: http
+      port: 3000
+      targetPort: 3000
+  selector:
+    app: zone-aware-routing-backend
+```
+{{% /tab %}}
+{{< /tabpane >}}
+
+#### Traffic Distribution
+{{< tabpane text=true >}}
+{{% tab header="Apply from stdin" %}}
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: zone-aware-routing-backend
+  labels:
+    app: zone-aware-routing-backend
+    service: zone-aware-routing-backend
+spec:
+  ports:
+    - name: http
+      port: 3000
+      targetPort: 3000
+  selector:
+    app: zone-aware-routing-backend
+  trafficDistribution: PreferClose
+EOF
+```
+{{% /tab %}}
+{{% tab header="Apply from file" %}}
+Save and apply the below service manifest to your cluster:
+```yaml
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: zone-aware-routing-backend
+  labels:
+    app: zone-aware-routing-backend
+    service: zone-aware-routing-backend
+spec:
+  ports:
+    - name: http
+      port: 3000
+      targetPort: 3000
+  selector:
+    app: zone-aware-routing-backend
+  trafficDistribution: PreferClose
+```
+{{% /tab %}}
+{{< /tabpane >}}
+
+
+### Example deployments and HTTPRoute
+Next apply the example manifests to create two Deployments and an HTTPRoute. For the test configuration one Deployment
+(zone-aware-routing-backend-local) includes affinity for EnvoyProxy Pods to ensure its Pods are scheduled in the same
+zone and the second Deployment (zone-aware-routing-backend-nonlocal) uses anti-affinity to ensure its Pods _don't_ 
+schedule to the same zone in order to demonstrate functionality. 
+```shell
+kubectl apply -f https://github.com/envoyproxy/gateway/releases/download/latest/zone-aware-routing.yaml -n default
```

Comment:
we need to confirm that this URL worked.

Reply:
This is actually wrong! Thanks for catching